includes/class-admin.php

```diff
@@ -469,4 +477,39 @@ public static function handle_bulk_request( $sendback, $action, $users ) {
 
 		return $sendback;
 	}
+
+	/**
+	 * Add ActivityPub infos to the dashboard glance items
+	 *
+	 * @param array $items The existing glance items.
+	 *
+	 * @return array The extended glance items.
+	 */
+	public static function dashboard_glance_items( $items ) {
+		\add_filter( 'number_format_i18n', '\Activitypub\custom_large_numbers', 10, 3 );
+
+		if ( ! is_user_disabled( get_current_user_id() ) ) {
+			$items['activitypub'] = sprintf(
+				'<a class="activitypub-followers" href="%s" title="%s">%s %s</a>',
```

Comment:
Personal pet-peeve of mine, I find statements like these easier to parse with numbered placeholders:

```suggestion
				'<a class="activitypub-followers" href="%1$s" title="%2$s">%3$s %4$s</a>',
```